openmetrics/README.md

```diff
@@ -22,7 +22,7 @@ For each instance the following parameters are required:
 
 | Parameter        | Description                                                                                                                                                                                                                                                              |
 | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `prometheus_url` | The URL where your application metrics are exposed by Prometheus (must be unique).                                                                                                                                                                                       |
+| `openmetrics_endpoint` | [Python3 only] The URL where your application metrics are exposed by Prometheus (must be unique). If running on Python 2 use the old `prometheus_url` configuration option name.                                                                                                                         |
```

Comment:
It's not only if running py2, it's if you were already using the old implementation and you want a reference to the old config